website_chd_product_configurator/static/src/css/website_chd_product_configurator.css

```diff
@@ -0,0 +1,55 @@
+<style type="text/css">
+    .divTable
+    {
```

Comment:
stick to one way of formatting you css brackets. I prefer the notation you use in this block